WyattHWilliams/react-challenge-project-jan-2022
# #4 Bugfix: include user in order form

## Changes
1. changed payload.login to payload.email in state handler

## Purpose
addresses [issue 4](https://github.com/Shift3/react-challenge-project-jan-2022/issues/4)

## Approach
1. by changing the state to get the correct info from the correct field. The state is used correctly by the order-form, previously the state was just receiving a param that didn't exist.

## Pre-Testing TODOs
1. login
2. create an order using the order form.

## Testing Steps
1. check that the data being passed into the POST request to the server includes the users email
2. check view orders list for ordered by: [users_email]

Closes #004
